.github/workflows/testing.yml

```diff
@@ -55,3 +55,5 @@ jobs:
         run: |
           cd app/
           python manage.py check
+      - name: Set up ChromeDriver
+        uses: nanasess/setup-chromedriver@v2.2.2
```

Comment:
I notice this image now, which I can't find on docker hub. Isn't there a more official image for selenium?

Reply:
Sorry, my bad, it is a docker action.

Reply:
For posterity: turns out this action wasn't doing anything anyway as it happens after the test